README.md

```
@@ -19,18 +21,28 @@ A web interface for chained notes.
+3. Start all the docker compose services:
 
    ```sh
-   bun run dev
+   # start the services
+   docker compose up
+
+   # start the services in the background
+   docker compose up --detach
    ```
 
+4. After all the services have started the frontend will be running at https://localhost.
```

Markdownlint

Bare URL used

(MD034, no-bare-urls)